README.md

```diff
@@ -62,14 +62,23 @@ To get started you will first need to satisfy the following pre-requisites:
 
 ## Deploy the Operator via ODH
 
+On a cluster with ODH installed, create a namespace where you would like to install DSPO:
 Deploy the following `DataScienceCluster`:
 
+```bash
+ODH_NS=opendatahub
+oc new-project ${ODH_NS}
+```
+
+Then deploy the following `DataScienceCluster` into the namespace created above:
+
 ```bash
 cat <<EOF | oc apply -f -
 kind: DataScienceCluster
 apiVersion: datasciencecluster.opendatahub.io/v1
 metadata:
   name: data-science-pipelines-operator
```

Comment:
it seems a little confusing that this would create a DataScienceCluster that is named DataSciencePipelinesOperator, empasis on *Cluster* and *Operator*, respectively

Reply:
Yeah, definitely.
Fixed.